war/src/main/java/io/lumeer/engine/controller/OrganizationFacade.java

```diff
@@ -0,0 +1,463 @@
+    *
+    * @param organizationId
+    *       id of the organization
+    * @param metaName
+    *       name of the meta attribute to update
+    * @param value
+    *       meta attribute value
+    */
+   public void updateOrganizationMetadata(final String organizationId, final String metaName, final String value) {
+      updateOrganizationMetadata(organizationId, new DataDocument(metaName, value));
+   }
+
+   /**
+    * Updates multiple organization metadata.
+    *
+    * @param organizationId
+    *       id of the organization
+    * @param meta
+    *       key-value pairs of metadata to update
+    */
+   public void updateOrganizationMetadata(final String organizationId, final DataDocument meta) {
+      dataStorage.updateDocument(LumeerConst.Organization.COLLECTION_NAME, meta, organizationIdFilter(organizationId));
+   }
+
+   /**
+    * Removes the specific organization metadata.
+    *
+    * @param organizationId
+    *       id of the organization
+    * @param metaName
+    *       name of the meta attribute to remove
+    */
+   public void dropOrganizationMetadata(final String organizationId, final String metaName) {
+      dataStorage.dropAttribute(LumeerConst.Organization.COLLECTION_NAME, organizationIdFilter(organizationId), metaName);
+   }
+
+   /**
+    * Creates new organization in the system database.
+    *
+    * @param organizationId
+    *       id of the new organization to create
+    * @param organizationName
+    *       name of the new organization to create
+    */
+   public void createOrganization(final String organizationId, final String organizationName) {
+      DataDocument document = new DataDocument(LumeerConst.Organization.ATTR_ORG_ID, organizationId)
+            .append(LumeerConst.Organization.ATTR_ORG_NAME, organizationName)
+            .append(LumeerConst.Organization.ATTR_ORG_DATA, new DataDocument())
+            .append(LumeerConst.Organization.ATTR_USERS, new DataDocument());
+      dataStorage.createDocument(LumeerConst.Organization.COLLECTION_NAME, document);
+   }
+
+   /**
+    * Renames a name of the given organization according to its id.
+    *
+    * @param organizationId
+    *       id of the given organization
+    * @param newOrganizationName
+    *       new name of the organization
+    */
+   public void renameOrganization(final String organizationId, final String newOrganizationName) {
+      DataDocument dataDocument = new DataDocument(LumeerConst.Organization.ATTR_ORG_NAME, newOrganizationName);
+      dataStorage.updateDocument(LumeerConst.Organization.COLLECTION_NAME, dataDocument, organizationIdFilter(organizationId));
+   }
+
+   /**
+    * Drops the organization according to its id.
+    *
+    * @param organizationId
+    *       id of the given organization to drop
+    */
+   public void dropOrganization(final String organizationId) {
+      dataStorage.dropDocument(LumeerConst.Organization.COLLECTION_NAME, organizationIdFilter(organizationId));
+   }
```

Comment:
We want here som checking if it can be achieved. But it needs other facades changes.